Assets/Scripts/Menu/MenuManager.cs

```diff
@@ -89,31 +121,47 @@ IEnumerator Start()
 			UpdateLocale();
 		}
 
+		public void ChangeScene() {
+			int index = 0;
+
+			for (int i = 0; i < allCupsAssets.cups.Length;i++) {
+				if (allCupsAssets.cups[i].cupID == PlayerPrefs.GetInt("CurrentCupID")) { 
+					index= i;
+					break;
+				}
+			}
+
+			PlayerPrefs.SetInt("CurrentSpeedway",0);
```

Comment:
Try to centralize player pref operations

Reply:
Corregido